app/views/categories/show.html.haml

```diff
@@ -0,0 +1,8 @@
+%article.video_category
+  %header
+    %h3= @category.name
+  .videos.row
+    - @category.videos.each do |video|
+      .video.col-sm-2
+        = link_to video, nil do
+          = image_tag(video.small_cover_url)
```

Comment:
Good job using image_tag :) A lot of times you will see it without brackets though :) pretty much like link_to helper.